src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -146,13 +146,16 @@ test_swim_status_handler(crt_rpc_t *rpc_req)
 	regex_t				 regex_dead;
 	static const char		*dead_regex = ".?0*1";
 	static const char		*alive_regex = ".?0*";
-	char				*swim_seq = malloc(MAX_SWIM_STATUSES);
+	char				swim_seq[MAX_SWIM_STATUSES];
+	size_t				sc = sizeof(char);
 
 	/* CaRT internally already allocated the input/output buffer */
 	e_req = crt_req_get(rpc_req);
 
 	if (swim_status_by_rank[e_req->rank] != NULL)
-		strcpy(swim_seq, swim_status_by_rank[e_req->rank]);
+		memcpy(swim_seq,
+		       swim_status_by_rank[e_req->rank],
+		       strlen(swim_status_by_rank[e_req->rank]) * sc);
```

Comment:
ideally we want to get strlen of swim status by rank * sc and then ensure its below MAX_SWIM_STATUSES